internal/kafka/kafka.go

```diff
@@ -171,11 +172,24 @@ func (b *kafkaBroker) Consume(ctx context.Context, topic string, handler func(ct
 			logger.Warn().Err(err).Msgf("Error when reading message: %s", err.Error())
 		} else {
 			logger.Trace().Bytes("payload", msg.Value).Msgf("Received message with key: %s", msg.Key)
+			ctx, err = ctxval.WithIdentityFrom64(ctx, returnHeaderValue(ctx, msg.Headers))
+			if err != nil {
+				logger.Trace().Msgf("Could not add identity to context: %s", err)
+			}
 			handler(ctx, NewMessageFromKafka(&msg))
 		}
 	}
 }
 
+func returnHeaderValue(ctx context.Context, headers []protocol.Header) string {
```

Comment:
naming is weird here, I'd love to name it `getIdentityHeaderValue` or something along those lines :)

Reply:
Absolutely, this should probably be an exported function so others can use this. Looks like this is a pattern, I suggest something like `LookupKafkaHeader`. We will likely need something similar for our generic message that would be `LookupGenericHeader`.

Reply:
@ezr-ondrej Regarding 1, I don't have a problem doing it but is it a bit inconsistent, because we do not have a function initAPI for the pbapi. 
Maybe we should add a ticket to unify that?

Reply:
My main point on 1 was: lets move as much code as we can into a package.
Maybe you're right, there is some leftover code in pbapi that could be moved, but if we get to the level of pbapi, and extract functions from main into package I'm good.
Tho pbapi initializes pretty standard stack, that if doesn't work, we will notice right away and it would mean some major library (like `chi`) is broken, here we are initializing custom stuff, so might be a good idea to test that and thus extract the init code to a package so it's testable as well.
But as I said, if we extract only the functions, and have tests for them I'm good ^_^